apps/07jquery-tic-tac-toe/script.js

```diff
@@ -2,4 +2,64 @@
 
 $(document).on('ready', function() {
   // Put app logic in here
+
+var playerTurn = 'X';
+var stopGame = false;
```

Comment:
1) Your `play` variable is not defined as a variable here. 

2) You declare stopGame as a variable here, thus the function with the same name throws an undefined error.